config/cruisecontrol.properties

```diff
@@ -165,7 +165,13 @@ num.proposal.precompute.threads=1
 zookeeper.connect=localhost:2181/
 
 # The max number of partitions to move in/out on a given broker at a given time.
-num.concurrent.partition.movements.per.broker=10
+num.concurrent.inter.broker.partition.movements.per.broker=2
+
+# The max number of partitions to move between disks within a given broker at a given time.
+num.concurrent.intra.broker.partition.movements.per.broker=2
```

Comment:
`num.concurrent.intra.broker.partition.movements.per.broker` -> `num.concurrent.intra.broker.partition.movements` -- i.e. if the config says `intra.broker`, it is already implied that it is `per.broker`.